### PR TITLE
NO-ISSUE: Synchronize From Upstream Repositories

### PR DESCRIPTION
**Pull request by tmshort**
The downstream repository has been updated through the following upstream commit:

| Date | Commit | Author | Message |
| -    | -      | -      | -       |
|2024-09-11 14:49:13|[operator-framework/operator-controller@9491b59](https://github.com/operator-framework/operator-controller/commit/9491b59b52cf2f34181078a5ac11be073e20a7e7)|Todd Short|Downgrade from 1.22.7 to 1.22.5 (#1252)|
||[upstream commit list](https://github.com/operator-framework/operator-controller/commits/9491b59b52cf2f34181078a5ac11be073e20a7e7)|||

The `vendor/` directory has been updated and the following commits were carried:

| Date | Commit | Author | Message |
| -    | -      | -      | -       |
|2024-08-30 00:02:12|[openshift/operator-framework-operator-controller@de07dac](https://github.com/openshift/operator-framework-operator-controller/commit/de07dac014291b896f37265af7d78d1d78ad8907)|dtfranz|UPSTREAM: <carry>: Add OpenShift specific files|
|2024-08-30 00:02:12|[openshift/operator-framework-operator-controller@1ea0bd3](https://github.com/openshift/operator-framework-operator-controller/commit/1ea0bd36b089f85204528a5cc5cb1d7b095134d5)|Alexander Greene|UPSTREAM: <carry>: Drop commitchecker|
|2024-08-30 00:02:13|[openshift/operator-framework-operator-controller@b707d7c](https://github.com/openshift/operator-framework-operator-controller/commit/b707d7c07e3fdf808bb74c7552b0e9671f77d9f7)|AOS Automation Release Team|UPSTREAM: <carry>: Updating ose-olm-operator-controller-container image to be consistent with ART Reconciling with https://github.com/openshift/ocp-build-data/tree/4022cd290f00a44d667dda03f2d78d84a488c7ed/images/ose-olm-operator-controller.yml|
|2024-08-30 00:02:13|[openshift/operator-framework-operator-controller@1b659a8](https://github.com/openshift/operator-framework-operator-controller/commit/1b659a8a1179c9f7a5672e2ebb4869a8f33245d4)|Mikalai Radchuk|UPSTREAM: <carry>: update owners|
|2024-08-30 00:02:14|[openshift/operator-framework-operator-controller@3d5a478](https://github.com/openshift/operator-framework-operator-controller/commit/3d5a478689ec8793e65bd27a5745fd98bf99e642)|Brett Tofel|UPSTREAM: <carry>: Add pointer to tooling README|
|2024-08-30 00:02:14|[openshift/operator-framework-operator-controller@868725a](https://github.com/openshift/operator-framework-operator-controller/commit/868725ae2edbb8d0bda1358e3ab7232fc7baa680)|Mikalai Radchuk|UPSTREAM: <carry>: Disable Validating Admission Policy APIs downstream|
|2024-08-30 00:02:14|[openshift/operator-framework-operator-controller@37f3bfb](https://github.com/openshift/operator-framework-operator-controller/commit/37f3bfb8e6d6da55a5a2ae53c3a6ec0b1dda7108)|AOS Automation Release Team|UPSTREAM: <carry>: Updating ose-olm-operator-controller-container image to be consistent with ART for 4.16 Reconciling with https://github.com/openshift/ocp-build-data/tree/6250d54c4686a708ca5985afb73080e8ca9a1f7f/images/ose-olm-operator-controller.yml|
|2024-08-30 00:02:15|[openshift/operator-framework-operator-controller@0f1acae](https://github.com/openshift/operator-framework-operator-controller/commit/0f1acae41ee195d693055474a4b3f4632669305d)|Mikalai Radchuk|UPSTREAM: <carry>: Enable Validating Admission Policy APIs downstream|
|2024-08-30 00:02:15|[openshift/operator-framework-operator-controller@23e9ae2](https://github.com/openshift/operator-framework-operator-controller/commit/23e9ae2e35c6cbc3c6231f02aaff22210197faf5)|Ilias Rinis|UPSTREAM: <carry>: manifests: set required-scc for openshift workloads|
|2024-08-30 00:02:16|[openshift/operator-framework-operator-controller@40e7e0b](https://github.com/openshift/operator-framework-operator-controller/commit/40e7e0b834f942efa2e234a3799a3407d443b76c)|AOS Automation Release Team|UPSTREAM: <carry>: Updating ose-olm-operator-controller-container image to be consistent with ART for 4.17 Reconciling with https://github.com/openshift/ocp-build-data/tree/4c1326094222f9209876f06833179a1b9178faf7/images/ose-olm-operator-controller.yml|
|2024-08-30 00:02:16|[openshift/operator-framework-operator-controller@1ba4dc4](https://github.com/openshift/operator-framework-operator-controller/commit/1ba4dc44e0f841a9774083b1e96fcf5395618489)|everettraven|UPSTREAM: <carry>: add everettraven to approvers+reviewers|
|2024-08-30 00:02:17|[openshift/operator-framework-operator-controller@0033ca7](https://github.com/openshift/operator-framework-operator-controller/commit/0033ca7ec1a67da13b7e6be3556a5a87976f3d6f)|everettraven|UPSTREAM: <carry>: add openshift kustomize overlay|

This pull request is expected to merge without any human intervention. If tests are failing here, changes must land upstream to fix any issues so that future downstreaming efforts succeed.

/cc @openshift/openshift-team-operator-runtime
/cc @openshift/openshift-team-operator-ecosystem

### CODE REVIEW

**Comment by grokspawn** [comment]
https://github.com/openshift/operator-framework-operator-controller/pull/135#issuecomment-2343792186
I didn't find any refs to these groups in openshift/release... not sure where they are coming from...

File: Makefile [hunk 1]
```diff
@@ -299,6 +299,15 @@ quickstart: $(KUSTOMIZE) manifests #EXHELP Generate the installation release man
 
 ##@ Docs
 
+.PHONY: crd-ref-docs
```
[the remark this answers is not in the document]
Comment (LalatenduMohanty) [reply Makefile:302]: yes, I do not any-other way to drop this change from downstream.

**Comment by tmshort** [comment]
> I didn't find any refs to these groups in openshift/release... not sure where they are coming from...

They are coming from the bumper itself.

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (tmshort) [reply Makefile:302]: Maybe, after this merges.

**Comment by grokspawn** [comment]
https://github.com/openshift/operator-framework-tooling/pull/50 to fix https://github.com/openshift/operator-framework-operator-controller/pull/135#issuecomment-2343792186

**Pull request merged**